docs/01-overview/main-areas/observability/README.md

```diff
@@ -2,20 +2,32 @@
 title: What is Observability in Kyma?
 ---
 
-## Observability tools
+The term "Observability" can be defined as a measure of how well the internal states of single components can be reflected by the application's external outputs. The insights that are exposed are called "telemetry" or "signals" and they are usually displayed in the form of metrics, traces, and logs. They can be exposed by employing modern instrumentation.
 
-Kyma comes with tools that give you the most accurate and up-to-date monitoring, logging and tracing data.
+Modern cloud applications consist of multiple components that you can scale and deploy individually, and are focused on one concern using the most fitting technology - essentially, they're distributed systems. Monitoring such an application is no longer an action you apply to a specific component; instead applications must actively expose insights. Monitoring becomes a joint aspect for all components, where every single component must actively expose its internal state.
```

Comment:
I still don't think this paragraph brings a lot of value to the document. It could be cut out and the document would be shorter and more concise without losing any information the user needs when reading about observability in Kyma.

`essentially, they're distributed systems` -> does it relate to these modern applications?

Maybe we can make this paragraph a bit more concise? I 
`Monitoring aims at covering all applications in distributed systems instead of focusing on a single component.`
Let's discuss this separately.

Reply:
Removed with https://github.com/kyma-project/kyma/pull/13119/commits/4e6932d2eff1afc834ed9a341ce6f0717e952f22